File: JAVA/Daily Flash Codes/Week2/Day3_Jan22/code/prog5.java
import java.io.*;

class Program {
	public static void main(String args[]) throws IOException{

		BufferedReader br = new BufferedReader (new InputStreamReader(System.in));

		System.out.println("Enter row :");
		int row = Integer.parseInt(br.readLine());
		
		for(int olc =0 ; olc<row ;olc++) {
				for(int ilc = 0; ilc<= olc; ilc++)	
						System.out.print(" * ");
				System.out.println();
		}

		

	}
}

File: C++/Daily Flash/week1/Day6/code_C++/prog5.cpp
/*
 * Program 5 : Write a Program to print following Pattern.
Output :
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
*
*/
#include <iostream>

int main() {
	int row;
	std::cout << "Enter row: ";
	std::cin >> row;

	for(int olc = 0 ;olc < row; olc++) {
		for(int ilc = 0; ilc < row ; ilc++)
			std::cout << " 0 ";
		std::cout << std::endl;
	}
	return 0;
}
